app/models/spree/gateway/stripe_connect.rb

```diff
@@ -43,6 +43,11 @@ def void(response_code, _creditcard, gateway_options)
         provider.void(response_code, gateway_options)
       end
 
+      def credit(money, _creditcard, response_code, gateway_options)
```

Comment:
I would have expected to name it `refund`. Is this part of some existing interface?

Reply:
Yep, I have no control over the method naming, this whole gateway class is basically just a thin wrapper of `ActiveMerchant::Billing::StripeGateway` used by `Spree::Payment::Processing`. There is a `refund` method in `StripeGateway`, but I think it does something different (maybe refunds the entire order?)

Reply:
It would be good to explain this as method documentation so the next dev doesn't wonder the same.

Reply:
Done